src/catalog/src/builtin.rs

```diff
@@ -7173,6 +7205,16 @@ ON mz_internal.mz_webhook_sources (id)",
     is_retained_metrics_object: true,
 };
 
+pub static MZ_CATALOG_EXTRACTOR_CONNECTION: Lazy<BuiltinConnection> =
+    Lazy::new(|| BuiltinConnection {
+        name: "mz_catalog_extractor_connection",
```

Comment:
Can we call this `mz_analytics` or `mz_analytics_catalog` to tie it to `mz_analytics` more directly?